script/server/config/file.conf

```diff
@@ -4,6 +4,14 @@ store {
   ## store mode: file、db、redis
   mode = "file"
```

Comment:
It is recommended to remove this configuration item, but it is compatible with the program.

Reply:
ok